README.md

```diff
@@ -94,38 +78,19 @@ would be represented as:
 ```json
 {
   ...
-  "proj:authority": "epsg",
-  "proj:code": 32659,
+  "proj:code": "EPSG:32659",
   ...
 }
 ```
 
-#### proj:authority
```

Comment:
I think we can keep this list for reference which authorities are commonly used. It should probably just be all in uppercase?